src/util/cklists.h

```diff
@@ -662,7 +662,7 @@ class CkCompactVec : private CkSTLHelper<T> {
     size_t len; // total number of used elements in block, including ones before offset
     size_t offset;      // global seqno of the first element in the block
     size_t lastnull;    // the array index of the biggest consecutive NULLs
-    void makeBlock(int blklen_,int len_,int offset_=0,int lastnull_=-1) {
```

Comment:
Can you elaborate on this change? This seems like it has the potential to be disruptive. Maybe create a separate PR?

Reply:
It was giving compiler warnings because `lastnull` is a `size_t` and some parts of the code are setting it to `-1`. But I agree and will create a separate PR.

Reply:
Moved to #3223